help/en/html/doc/Technical/IntelliJ.shtml

```diff
@@ -492,82 +496,158 @@
           <li>Click "OK" to complete the configuration setup.</li>
           </ul>
         <li>To run your first debug session, start by opening a .java file and setting a Breakpoint on a line in your
         code by clicking in the left margin of the central edit pane:<br>
-        <img src="images/IntelliJ/IjDebug5Breakpoint.png" width="302" height="184"></li>
+        <img src="images/IntelliJ/IjDebug5Breakpoint.png" width="302" height="184" alt="IntelliJ
+        Debug Breakpoint in margin"></li>
         <li>We're now ready to start a Debug session by clicking the Bug button at the top right, making sure our
         "JMRI debug" configuration is selected in the drop down to the left of it:<br>
-        <img src="images/IntelliJ/IjDebug4StartDebug.png" width="272" height="79">
+        <img src="images/IntelliJ/IjDebug4StartDebug.png" width="272" height="79" alt="IntelliJ
+        Debug select your config set">
         </li>
         <li>After a short while, at the bottom of the main IntelliJ IDE interface, the Debug pane appears, stopping
         just before your Breakpoint (or some previous error):<br>
-        <img src="images/IntelliJ/IjDebug6DebugInterface.png" width="782" height="398"></li>
+        <img src="images/IntelliJ/IjDebug6DebugInterface.png" width="391" height="199" alt="IntelliJ
+        Debug code execution stopped at Breakpoint"></li>
         <li>Inspect the current values of the active variables, and click the Step Into button to execute 1 line of
         code:<br>
-        <img src="images/IntelliJ/IjDebug7StepInto.png" width="118" height="49"></li>
+        <img src="images/IntelliJ/IjDebug7StepInto.png" width="118" height="49" alt="IntelliJ
+        Debug Step Into buttons"></li>
       </ol>
       <p>A further explanation of this pane and debugging in general can be found in the
-      <a href="https://www.jetbrains.com/help/idea/debugging.html">IntelliJ Help</a>.<br>
+      <a href="https://www.jetbrains.com/help/idea/debugging.html">IntelliJ Online Help</a>.<br>
       If your code has not been compiled before debugging, the source and class files might be out of sync, so be
       sure to build first. You could add that to your Debug Configuration too.</p>
+      <p>Besides Debug, there's also a Coverage option available in the Debug menu and tool bar.</p>
+
+      <a name="findbugs" id="findbugs"></a>
+      <h2>Running FindBugs</h2>
+        <ol>
+        <li>Confirm that the class and output paths have been <a href="#debug">set up correctly</a> for the
+          compiler.</li>
+        <li>Install the IntelliJ FindBugs 1.01 plug-in. If your Findbugs installation is correct, the FindBugs tab at
+          bottom left will show, a FindBugs item will show up at the bottom of the <b>Analyze</b> menu and the path to
+          FindBugs 3.01 in your <code>local.properties</code> file will match the place where the
```

Comment:
Please note JMRI stopped using FindBugs in #4760 since FindBugs was abandoned upstream. The JMRI local.properties should *not* be pointing to a FindBugs installation, but to a SpotBugs installation, unless you are referring to a local.properties other than that used by ant.

Reply:
OK, thanks, will update the page. This means that we shouldn't use the FindBugs Ij plugin. Have you looked at the SpotBugs plugin ideas circulating?

Reply:
The IntelliJ FindBugs plugin developer is [planning on migrating the plugin to SpotBugs](https://github.com/andrepdo/findbugs-idea/issues/265).